src/VernaForm.tsx

```diff
@@ -1,31 +1,25 @@
+import { useVerna } from './VernaContextProvider';
 import Form from '@rjsf/core';
-import type { JSONSchema7 } from 'json-schema';
+import RenderFieldTemplate from './RenderMethods/RenderFieldTemplate';
 
-const schema: JSONSchema7 = {
-  title: 'A registration form',
-  description: 'A simple form example.',
-  type: 'object',
-  required: ['firstName', 'lastName'],
-  properties: {
-    firstName: {
-      type: 'string',
-      title: 'First name',
-      default: 'Richie',
-    },
-    lastName: {
-      type: 'string',
-      title: 'Last name',
-    },
-    phone: {
-      type: 'string',
-      title: 'Phone',
-      minLength: 10,
-    },
-  },
-};
+interface VernaFormProperties {
+  onSubmit?: () => void;
```

Comment:
The function signature is not correct.
```suggestion
  import { type FormProps } from '@rjsf/core';
  [...]
  onSubmit?: FormProps<unknown>['onSubmit'];
```